### DISCUSSION TITLE
Collapse Docs are missing in v2

### DISCUSSION THREAD

**Discussion by merlinstardust** [chakra-ui/chakra-ui#8691] [General]
### Description

When I go to the v2 Chakra UI docs, I expect to find documentation on the Collapse element.

![image](https://github.com/chakra-ui/chakra-ui/assets/421960/c8ce0512-7975-4e36-b0f0-03a0dd08fef9)

### Link to Reproduction

https://v2.chakra-ui.com/docs/components

### Steps to reproduce

1. Go to https://v2.chakra-ui.com/docs/components
2. Search for Collapse
3. See that no such docs exist

### Chakra UI Version

2.8.2

### Operating System

- [X] macOS
- [X] Windows
- [X] Linux

**Comment by segunadebayo** [comment]
v2 Collapse is `Collapsible` in v3. See https://chakra-ui.com/docs/components/collapsible
